content/news/026/index.md

```diff
@@ -129,6 +149,30 @@ or [join the next meeting][join].
 
 ## Bonus
 
+### Rust Graphics Meetup 1
+
+It happened! Check out the talks:
```

Comment:
It might be worth adding a line or two about what the Graphics Meetup is, since with you leading it, I think most people assume it's "talks about wgpu and friends" :P